src/controllers/Cart.controller.ts

```diff
@@ -0,0 +1,92 @@
+import { Request, Response, NextFunction } from 'express-serve-static-core';
+import { AppDataSource } from '../config/database';
+import { Cart } from '../entities/Cart';
+import { CartItem } from '../entities/CartItem';
+import { Product } from '../entities/Product';
+import {generatePaginationLinks} from '../untils/pagenation'
+
+const cartRepository = AppDataSource.getRepository(Cart);
+const cartItemRepository = AppDataSource.getRepository(CartItem);
+const productRepository = AppDataSource.getRepository(Product);
+
+
+
+export const getUserCart = async (userId: number,res: Response) => {
```

Comment:
method dùng trong module thì ko cần export e nhé
```suggestion
const getUserCart = async (userId: number,res: Response) => {
```